### PR TITLE
connect to Astra

### PR DESCRIPTION
**Pull request by patriknw**
I tried using DataStax Astra from GKE and it worked great. Not intended to be merged. Just showing what is needed.

Also requires the CassandraSession config changes from https://github.com/akka/akka-platform-guide/pull/201.

I used the script in #203 for creating the tables.

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/src/main/scala/shopping/cart/AstraSessionProvider.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class AstraSessionProvider(system: ActorSystem, config: Config) extends CqlSessionProvider {
+  override def connect()(implicit ec: ExecutionContext): Future[CqlSession] = {
+    val driverConfig = CqlSessionProvider.driverConfig(system, config)
+    val driverConfigLoader = DriverConfigLoaderFromConfig.fromConfig(driverConfig)
+    CqlSession
+      .builder()
+      .withConfigLoader(driverConfigLoader)
+      .withCloudSecureConnectBundle(
+        getClass.getClassLoader.getResourceAsStream("cassandra-astra-secure-connect-akka-patriknw.zip"))
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/src/main/scala/shopping/cart/AstraSessionProvider.scala:21]: the zip comes from the secure connect bundle from https://docs.astra.datastax.com/docs/connecting-to-your-database-with-the-datastax-java-driver
I'm not sure if it contains any secrets (not included in this PR anyway).

**Review by chbatey — COMMENTED** [review]
Nice, trying to connect to any of the other fake cassandras has always been quite a lot of effort

**Review by ignasi35 — APPROVED** [review]
Oh, surprisingly simple. Nice!

**Comment by patriknw** [comment]
served it's purpose as a spike

**Pull request closed without merging**